core/src/lib.rs

```diff
@@ -35,6 +35,8 @@
 //!   define how to upgrade each individual substream to use a protocol.
 //!   See the `upgrade` module.
 
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
These seems like it is a left over from another attempt at solving this.

Reply:
No I don't think so. This turns the required feature gate on. I.e. `#[doc(cfg(...))]`.

Reply:
Ah yes, my bad! Thanks for clarifying!